edxval/tests/test_api.py

```diff
@@ -939,14 +925,16 @@ def setUp(self):
         # create internal video transcripts
         transcript_data = dict(constants.VIDEO_TRANSCRIPT_CIELO24, video=video)
         transcript_data.pop('video_id')
+        transcript_data.pop('file_data')
         VideoTranscript.objects.create(**transcript_data)
         transcript_data = dict(constants.VIDEO_TRANSCRIPT_3PLAY, video=video)
         transcript_data.pop('video_id')
+        transcript_data.pop('file_data')
```

Comment:
we can create a helper method to omit un-needed attributes, the above three lines would become one.
```bash
def omit_attrs(dict, attrs_to_omit=[]):
    return {attr: value for attr, value in dict.iteritems() if attr not in attrs_to_omit}
```